### ISSUE TITLE
Update Vocabulary Latest UPDATE on every change in vocabulary

### ISSUE THREAD

**Issue by acumarav** [OHDSI/Athena#164]
Vocabulary's Latest Update should change on every change in vocabulary, source and target

**Comment by acumarav** [comment]
As per discussion with @Alexdavv - the field reflects changes in the source vocabulary and it's behavior should not be changed.
The optimal change will be to add one more field which will be reflecting both source and output changes. After that Vocabularies update email notifications functionality can be switch to that new field

**Comment by mik-ohdsi** [comment]
During the release process, hand over the latest update date (and create a new field “source_version_date” in vocabulary_conversion table and repurpose the latest_update field). This would also cover relationship updates including "to or from" changes for the respective vocabulary (which are very frequent). 
![vocab_release_date](https://user-images.githubusercontent.com/60365423/122787308-7d6ad900-d2b5-11eb-8f2b-92c4d6c0ea0b.png)
(is it possible to get little information flyouts / hover tooltips to explain the purpose of the respective column?)

**Comment by Alexdavv** [comment]
@mik-ohdsi @cgreich What about the changes in ancestry? Let’s discuss it also.

**Comment by cgreich** [comment]
Good point. Yes, they should trigger an OHDSI release as well to all vocabularies that are touched by a change. Which means, that ATC will get a time stamp if something changes in RxNorm, even though that change does not affect any ATC concept or any relationship of an ATC concept. Thoughts?
